templates/node--funding--full.html.twig

```diff
@@ -0,0 +1,211 @@
+
+{%
+set classes = [
+'node',
+'node--type-' ~ node.bundle|clean_class,
+node.isPromoted() ? 'node--promoted',
+node.isSticky() ? 'node--sticky',
+not node.isPublished() ? 'node--unpublished',
+view_mode ? 'node--view-mode-' ~ view_mode|clean_class,
+'clearfix',
+]
+%}
+{{ attach_library('classy/node') }}
+{{ attach_library('unl_fourone/funding') }}
+
+{%
+set full_description, funding_amount, funding_periods, application_due_date, contact_name, contact_phone_number, contact_email, eligibility, how_to_apply, templates, resources_documents, resources_external_links, apply_link, application_form, applicable_to =
+    content.field_funding_full_description, content.field_funding_amount, content.field_funding_periods, content.field_funding_app_due_date, content.field_funding_contact_name, content.field_funding_contact_phone, content.field_funding_contact_email, content.field_funding_eligibility, content.field_funding_how_to_apply, content.field_funding_templates, content.field_funding_resources_docs, content.field_funding_resources_links, content.field_funding_apply_link, content.field_funding_application_form, content.field_applicable_to
```

Comment:
I'm not a big fan of doing this because it will make future maintenance (searching, auto completion, etc) more difficult